app/src/main/java/org/mozilla/tv/firefox/telemetry/TelemetryIntegration.kt

```diff
@@ -146,22 +153,31 @@ open class TelemetryIntegration protected constructor(
         resetSessionMeasurements(context)
     }
 
+    // EXT to add events to pocket ping (independently from mobile_events)
+    fun TelemetryEvent.queueInPocketPing() = (TelemetryHolder.get()
+            .getPingBuilder(TelemetryPocketEventPingBuilder.TYPE) as TelemetryPocketEventPingBuilder)
```

Comment:
One more nit: please add a check here for whether data collection is enabled/disabled, similar to the other event queuing like https://github.com/mozilla-mobile/android-components/blob/master/components/service/telemetry/src/main/java/org/mozilla/telemetry/Telemetry.java#L95